### PR TITLE
Added prometheus metrics

### PR DESCRIPTION
**Pull request by Iodine98**

### CODE REVIEW

**Review by kandrio — COMMENTED** [review]
What is the purpose of this `Made progress somewhat.` commit?

**Review by kandrio — COMMENTED** [review]
When I take a look at the `Added the prometheus metrics.` commit diff, I don't quite understand what's going on. All lines look changed but all lines remain the same as well.

**Review by kandrio — COMMENTED** [review]
This `Used prometheus flask exporter...` commit is awesome! Awesome finding. It looks so much better now.

File: model_server.py [hunk 1]
```diff
@@ -19,7 +19,7 @@
 )
 
 predictions_counter = prom_metrics.counter(
-    'predictions', 'The number of predictions.',
+    'predictions', 'The number of predictions.', labels={'count': 1}
```
Comment (kandrio) [inline model_server.py]: What does the `count`: 1 mean exactly?

**Review by kandrio — COMMENTED** [review]
What exactly does the Prometheus client do?

File: pyproject.toml [hunk 2]
```diff
@@ -4,6 +4,7 @@ version = "0.1.0"
 description = "This repo serves the model so that it can be queried."
 authors = ["Your Name <you@example.com>"]
 readme = "README.md"
+packages = [{include = "model-service"}]
```
Comment (kandrio) [inline pyproject.toml]: Again, I notice you add that and then you remove it.

File: model_server.py (hunk 1 above)
Comment (Iodine98) [reply model_server.py]: I wasn't sure if all Prometheus metrics need labels by default, so I made a dict with a static key-value pair. I don't think it's necessary though.

**Comment by Iodine98** [comment]
> What is the purpose of this `Made progress somewhat.` commit?

It's probably a bad habit, but whenever I try to figure out stuff and I don't concretely know what I implemented, I created these kinds of commits before closing my laptop so that I always have a state to go back to. 

Any tips on how to improve? @kandrio

**Comment by Iodine98** [comment]
> When I take a look at the `Added the prometheus metrics.` commit diff, I don't quite understand what's going on. All lines look changed but all lines remain the same as well.

I think my git client messed things up when I committed through WSL vs PyCharm because of different line endings in Windows and Linux. I'll try to pay attention to it next time.

**Comment by Iodine98** [comment]
In short I added the metrics in the `model-service` repo by trying out two different libraries where I eventually settled on the last library, since that is the officially supported library by Prometheus whereas the first one was a project forked from this and had odd features which were poorly documented.

**Pull request merged**